### ISSUE TITLE
关于文章中with perceptual loss，以及数据集中GT Mean

### ISSUE THREAD

**Issue by climberwu** [Fediory/HVI-CIDNet#39]
请问，论文中的with perceptual loss是如何使用呢，数据集中GT Mean是在哪里获取呢？？？

**Comment by Fediory** [comment]
请参考measure.py中的这段代码。
`if use_GT_mean:
            mean_restored = cv2.cvtColor(im1, cv2.COLOR_RGB2GRAY).mean()
            mean_target = cv2.cvtColor(im2, cv2.COLOR_RGB2GRAY).mean()
            im1 = np.clip(im1 * (mean_target/mean_restored), 0, 255)`
这只是一个measure trick，目的是为了将图像亮度与GT矫正后，来对比颜色细节和结构信息。

**Comment by Fediory** [comment]
对于Perceptual Loss，请参考Train中的
`P_loss = PerceptualLoss({'conv1_2': 1, 'conv2_2': 1,'conv3_4': 1,'conv4_4': 1}, perceptual_weight = P_weight ,criterion='mse').cuda()`

**Comment by Fediory** [comment]
因为LOLv1数据集包含更少的图片，因此一些细微的亮度差异会导致指标大幅波动。我们更建议在LOLv1上使用GT mean，而其它数据集中保持原本的测试标准。

**Comment by Fediory** [comment]
GT mean是一个测试指标，所以只需要在measure.py中使用GT mean就可以。
`python measure.py --lol --use_GT_mean`
